controllers/account/account_controller_test.go

```diff
@@ -1415,4 +1420,167 @@ var _ = Describe("Account Controller", func() {
+						return []string{account.Status.State, account.Status.SupportCaseID}
+					}).Should(Equal([]string{AccountReady, "123456"}))
+				})
+				It("creates a servicequota case for each defined quota", func() {
+					subClient := mock.NewMockClient(ctrl)
+					AssumeRole = func(r *AccountReconciler,
+						reqLogger logr.Logger,
+						currentAcctInstance *awsv1alpha1.Account,
+						awsSetupClient awsclient.Client,
+						roleToAssume string,
+						ccsRoleID string) (awsclient.Client, *sts.AssumeRoleOutput, error) {
+						return subClient, &sts.AssumeRoleOutput{}, nil
+					}
+					// Reconciliation loop 1
+					mockAWSClient.EXPECT().CreateCase(gomock.Any()).Return(&support.CreateCaseOutput{
+						CaseId: aws.String("123456"),
+					}, nil)
+					mockAWSClient.EXPECT().DescribeCases(gomock.Any()).Return(&support.DescribeCasesOutput{
+						Cases: []*support.CaseDetails{
+							{
+								CaseId: aws.String("123456"),
+								Status: aws.String("resolved"),
+							},
+						},
+					}, nil)
+					subClient.EXPECT().ListRequestedServiceQuotaChangeHistoryByQuota(gomock.Any()).Return(&servicequotas.ListRequestedServiceQuotaChangeHistoryByQuotaOutput{
+						RequestedQuotas: []*servicequotas.RequestedServiceQuotaChange{},
+					}, nil)
+					subClient.EXPECT().GetServiceQuota(gomock.Any()).Return(&servicequotas.GetServiceQuotaOutput{
+						Quota: &servicequotas.ServiceQuota{
+							QuotaCode: aws.String(string(awsv1alpha1.VCPUQuotaCode)),
+							Value:     aws.Float64(0),
+						},
+					}, nil)
+					subClient.EXPECT().RequestServiceQuotaIncrease(gomock.Any()).Return(&servicequotas.RequestServiceQuotaIncreaseOutput{
+						RequestedQuota: &servicequotas.RequestedServiceQuotaChange{
+							CaseId: aws.String("234567"),
+						},
+					}, nil)
+					// Reconciliation loop 2
+					mockAWSClient.EXPECT().DescribeCases(gomock.Any()).Return(&support.DescribeCasesOutput{
+						Cases: []*support.CaseDetails{
+							{
+								CaseId: aws.String("123456"),
+								Status: aws.String("resolved"),
+							},
+						},
+					}, nil)
+					// The quota now matches the requested value the case is finished
+					subClient.EXPECT().GetServiceQuota(gomock.Any()).Return(&servicequotas.GetServiceQuotaOutput{
+						Quota: &servicequotas.ServiceQuota{
+							QuotaCode: aws.String(string(awsv1alpha1.VCPUQuotaCode)),
+							Value:     aws.Float64(100),
+						},
+					}, nil)
+					Eventually(func() []string {
+						r.HandleNonCCSPendingVerification(nullLogger, account, mockAWSClient)
+						return []string{account.Status.State, account.Status.SupportCaseID}
+					}).Should(Equal([]string{AccountReady, "123456"}))
+					var k8sAccount awsv1alpha1.Account
+					r.Client.Get(context.TODO(), types.NamespacedName{
+						Namespace: TestAccountNamespace,
+						Name:      TestAccountName,
+					}, &k8sAccount)
+					Expect(k8sAccount.Status.State).To(Equal(AccountReady))
+				})
+			})
+		})
+	})
+
+	AfterEach(func() {
```

Comment:
Probably move this back up to where it was to be more in line with the rest of our tests